source/server/admin/admin.h

```diff
@@ -211,6 +211,10 @@ class AdminImpl : public Admin,
   const HttpConnectionManagerProto::ProxyStatusConfig* proxyStatusConfig() const override {
     return proxy_status_config_.get();
   }
+  Http::HeaderValidatorPtr makeHeaderValidator(Http::Protocol, StreamInfo::StreamInfo&) override {
+    // TODO(yanavlasov): admin interface should use the default validator
```

Comment:
should this ENVOY_BUG?  seems like it's pretty unsafe to do admin requests until this is fixed

Reply:
Done

Reply:
Had to undo ENVOY_BUG as it crashed all integration tests with UHV enabled. This item is on my internal OKR tracker and will not be dropped.